pkg/scaffold/v2/internal/string_utils.go

```diff
@@ -115,3 +111,32 @@ func InsertStringsInFile(path string, markerAndValues ...string) error {
 
 	return err
 }
+
+// filterExistingValues removes the single-line values that already exists in
+// the given reader. Multi-line values are ignore currently simply because we
+// don't have a use-case for it.
+func filterExistingValues(r io.Reader, markerAndValues map[string][]string) error {
+	scanner := bufio.NewScanner(r)
+	for scanner.Scan() {
+		line := scanner.Text()
+		for marker, vals := range markerAndValues {
+			for i, val := range vals {
+				if strings.TrimSpace(line) == strings.TrimSpace(val) {
+					markerAndValues[marker] = removeElementAt(vals, i)
+				}
+			}
+		}
+	}
+	if err := scanner.Err(); err != nil {
+		return err
+	}
+	return nil
+}
+
+// removeElementAt removes the element at index i from a.
+func removeElementAt(a []string, i int) []string {
```

Comment:
This function can be one single line according to [this](https://github.com/golang/go/wiki/SliceTricks#delete).

Reply:
Thanks for the tip. Done :)